<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgAVEditOutput</class>
 <widget class="QDialog" name="DlgAVEditOutput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>351</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="m_pBtnClose">
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="m_pGroupBoxAudio">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>音频参数</string>
   </property>
   <widget class="QLabel" name="m_pLabelTxtSampleRate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>采样率：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pCmbbxSampleRate">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>44100</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>48000</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="m_pLabelTxtAChannel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>声道：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pCmbbxChannel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>单声道</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>双声道</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="m_pGroupBoxVideo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>129</y>
     <width>431</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>视频参数</string>
   </property>
   <widget class="QLabel" name="m_pLabelTxtSize">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>分辨率：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pCmbSize">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1920x1080</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1280x720</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>720x480</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QProgressBar" name="m_progressBarEncode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>421</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnStop">
   <property name="geometry">
    <rect>
     <x>394</x>
     <y>290</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnPause">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnStart">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>290</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_pBtnClose</sender>
   <signal>clicked()</signal>
   <receiver>DlgAVEditOutput</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
